opensaml/3.3.1.wso2v1/pom.xml

```diff
@@ -0,0 +1,1155 @@
+                        <Private-Package>
+                        </Private-Package>
+                        <Import-Package>
+                            !org.opensaml.*,
+                            !net.shibboleth.*,
+                            !org.cryptacular.*,
+                            javax.crypto.*;version="${javax.crypto.version}",
+                            javax.net.*;version="${javax.net.version}",
+                            javax.security.*;version="${javax.security.version}",
+                            javax.servlet.jsp.*;version="${javax.servlet.jsp.version}",
+                            javax.servlet.*;version="${javax.servlet.version}",
+                            javax.xml.*;version="${javax.xml.version}",
+                            org.apache.axiom.om.impl.dom.jaxp;version="${axiom.osgi.version.range}",
+                            org.apache.commons.beanutils;version="${org.apache.commons.beanutils.version}";resolution:=optional,
+                            org.apache.commons.configuration;version="${org.apache.commons.configuration.version}";resolution:=optional,
+                            org.apache.commons.fileupload.*;version="${org.apache.commons.fileupload.version}";resolution:=optional,
+                            org.apache.commons.collections.iterators;version="${org.apache.commons.collections.iterators.version}";
+                            resolution:=optional,
+                            org.apache.log4j.*;version="${apache.log4j.version}";resolution:=optional,
+                            org.apache.xml.security.*;version="${org.apache.xml.security.version}";resolution:=optional,
+                            org.apache.velocity.*;version="${org.apache.velocity.version}";resolution:=optional,
+                            org.joda.time.*;version="${joda-time.orbit.version.range}",
+                            org.bouncycastle.*;version="${org.bouncycastle.version}";resolution:=optional,
+                            org.slf4j.*;version="${org.slf4j.version}",
+                            org.w3c.*;version="${org.w3c.version}";resolution:=optional,
+                            org.xml.sax.*;version="${org.xml.sax.version}";resolution:=optional,
+                            junit.framework;version="${junit.framework.version}";resolution:=optional,
+                            junit.textui;version="${junit.textui.version}";resolution:=optional,
+                            bsh;version="${bsf.version}";resolution:=optional,
+                            nu.xom;version="${nu.xom.version}";resolution:=optional,
+                            org.owasp.esapi;resolution:=optional,
+                            org.owasp.validator.html;version="${org.owasp.validator.html.version}";resolution:=optional,
+                        </Import-Package>
+                        <Embed-Dependency>esapi;scope=compile|runtime;inline=false,commons-codec;inline=true,</Embed-Dependency>
```

Comment:
Check if we need to embed this

Reply:
Yes caused this is not found in components/dropins or components/plugins in product-is